lib/types/string.js

```diff
@@ -742,7 +742,8 @@ module.exports = Any.extend({
 internals.addressOptions = function (options) {
 
     if (!options) {
-        return options;
+        assert(internals.tlds, 'Built-in TLD list disabled');
```

Comment:
I noticed that all the references in code to `internals.tlds` include this `assert` to ensure the attribute is defined. It's reasonable to assume that if the default behavior changes, this `assert` could be removed. Otherwise, it seems justified to keep it, what do you think?

Reply:
I removed the assertion. Let me know  what you think about it